tools/src/prebuilds/XcodeGen.ts

```diff
@@ -212,3 +213,44 @@ function podNameToBundleId(podName: string): string {
     .replace(/(\_|[^\w\d\.])+/g, '.')
     .replace(/\.*([A-Z]+)/g, (_, p1) => `.${p1.toLowerCase()}`);
 }
+
+/**
+ * Generate custom modulemap for expo-modules-core where requiring React-Core as modular headers
+ */
+export async function generateExpoModulesCoreModulemapAsync(destDir: string): Promise<string> {
+  const expoModulesCoreUmbrellaHeader = path.join(
+    PODS_PUBLIC_HEADERS_DIR,
+    'ExpoModulesCore',
+    'ExpoModulesCore-umbrella.h'
```

Comment:
It's also used more than once, so make it a global constant or a constant just here in this function 😉